src/components/dao/dao.js

```diff
@@ -121,7 +124,7 @@ export default class Dao extends Component {
                                     job_bond: infos.policy.proposal_bond
                                 }
                             })),
-                            deposit: depo.toString(),
+                            deposit: depo.toFixed(),
```

Comment:
`toString()` should be the correct thing here (?)

Reply:
`depo` is of type `Big`, for which I prefer `toFixed()` as its behavior is predictable. `toString()` returns scientific notation in some cases so it's better to stay away from it

Reply:
Be aware that `toFixed()` will remove decimal places. This might be fine in some cases but certainly not all. IMO just always using `toString()` with a large `Big.PE` is fine.